packages/server-web/src/client/component/Profile.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div id="profile" v-if="isAuthenticated">
+    <button id="profile-button" v-on:click="menuCollapsed=!menuCollapsed"><i class="fas fa-user"></i></button>
+    <div style="clear: both;"></div>
+    <div id="menu" v-if="!menuCollapsed">
+      <ul>
+        <li><button v-on:click="showPasswordChanger=true">Change password</button></li>
+        <li><router-link tag="button" v-if="isAuthenticated" to="/login" v-on:click.native="logout()" replace>Logout</router-link></li>
+      </ul>
+      <div style="clear: both;"></div>
+    </div>
+    <password-changer v-if="showPasswordChanger" @close="showPasswordChanger = false" />
+  </div>
+</template>
+
+<script>
+import PasswordChanger from './PasswordChanger.vue';
+export default {
+  name: "Profile",
+  components: {
+    "password-changer": PasswordChanger
+  },
+  data() {
+    return {
+      menuCollapsed: true,
+      showPasswordChanger: false
+    }
+  },
+  methods: {
+    showPasswordPopup() {
+      alert("CHANGE THE PASSWORDZ");
+    },
+    logout() {
+      this.$http
+        .post("/logout", {})
+        .then(function(response) {
+          this.$store.commit('isAuthenticated', false);
+          this.$store.commit('isLoading', false);
+          this.$router.replace({ name: "login" });
+        })
+        .catch(function(err) {
+          console.log(err);
```

Comment:
The logout also destroys the session on the server. If it can't do that (because internet connection is gone, for example)... That being said, it's probably no problem to set the state to false, you're right.